ui/base-tool/src/main/kotlin/org/cru/godtools/base/tool/ui/settings/ToolOptionsSettingsBottomSheetDialogFragmentDataModel.kt

```diff
@@ -21,7 +21,7 @@ import org.cru.godtools.model.Language.Companion.sortedByDisplayName
 
 @HiltViewModel
 @OptIn(ExperimentalCoroutinesApi::class)
-class SettingsBottomSheetDialogFragmentDataModel @Inject constructor(
+class ToolOptionsSettingsBottomSheetDialogFragmentDataModel @Inject constructor(
```

Comment:
I commented this elsewhere, but this name is redundant, choose either `Settings` or `Options`, but not both

Reply:
my preference is to leave it as `Settings` since that is what it was before